examples/cc-template/WORKSPACE

```diff
@@ -0,0 +1,36 @@
+# load the http_archive rule itself
```

Comment:
Once you rebase on master you can find an easy example of how to test this on CI [here](https://github.com/tweag/rules_nixpkgs/blob/691582fd6712502b38e7ef6a488203edc79143b8/.github/workflows/workflow.yaml#L122-L126).